docs/sources/features/explore/index.md

```diff
@@ -0,0 +1,104 @@
+
+## Split and Compare
+
+The Split feature is an easy way to compare graphs and tables side-by-side or to look at related data together on one page. Click the split button to duplicate the current query and split the page into two side-by-side queries. It is possible to select another datasource for the new query which for example, allows you to compare the same query for two different servers or to compare the staging environment to the production environment.
+
+You can close the newly created query by clicking on the Close Split button.
+
+## Prometheus-specific Features
+
+The first version of Explore features a custom querying experience for Prometheus. When a query is executed, it actually executes two queries. One for the graph and an Instant Query for the table. An Instant Query returns the last value for each time series e.g. if it the query results returns 
+
+### Metrics Explorer
+
+On the left-hand side of the query field is a `Metrics` button, clicking on this opens the Metric Explorer. This shows a hierarchical menu with metrics grouped by their prefix. For example, all the Alert Manager metrics will be grouped under the `alertmanager` prefix. This is a good starting point if you just want to explore which metrics are available.
+
+### Query Field
+
+The Query field supports autocomplete for metric names, function and works mostly the same way as the standard Prometheus query editor. Press the enter key to execute a query.
+
+The autocomplete menu can be trigger by pressing Ctrl + Space. The Autocomplete menu contains a new History section with a list of recently executed queries.
+
+Suggestions can appear under the query field - click on them to update your query with the suggested change.
+
+- For counters (monotonously increasing metrics), a rate function will be suggested.
+- For buckets, a histogram function will be suggested.
+- For recording rules, possible to expand the rules.
+
+### Table Filters
+
+Click on the cell links in the Table panel to add filters to the query expression. This works with multiple queries too - the filter will be added for all the queries.
+
+### Logs Integration
+
+For Grafana 6.0, the first log integration is for the new open source log aggregation system from Grafana Labs - Loki. Loki is designed to be very cost effective, as it does not index the contents of the logs, but rather a set of labels for each log stream. The logs from Loki are queried in a similar to metric queries in Prometheus. It uses labels to group log streams which can be made to match up with your Prometheus labels. Read more about Loki here or the Grafana Labs hosted variant: Grafana Cloud Logs.
+
+If you switch from a Prometheus query to a logs query (you can do a split first to have your metrics and logs side by side) then it will keep the labels from your query and use those to query the log streams. For example, the following Prometheus query:
```

Comment:
...will keep the labels from your query that exist in the logs, and use those...

Reply:
Ha, didn't know that it was that smart. Thanks.